deblur/workflow.py

```diff
@@ -434,7 +441,7 @@ def op(x): return x in aligned_seq_ids
                 badseqs += 1
     logger.info('total sequences %d, passing sequences %d, '
                 'failing sequences %d' % (totalseqs, okseqs, badseqs))
-    return output_fp
+    return output_fp, okseqs
```

Comment:
to verify, were all instances of the calls to this method updated with respect to the API change?